src/data_import/mongodb/gene_functions.py

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Looks OK to me. Can you add docstrings so we all know how to use these functions? Also try to avoid names with the generic `get_` prefix in favor of more descriptive naming. LMK know if you would like some ideas on this.

A fundamental rule I follow is:

>  Function names should be verbs if the function changes the state of the program, and nouns if they're used to return a certain value.